src/option_parser.c

```diff
@@ -249,25 +284,10 @@ int load_ini_file(FILE * fp)
                 }
 
                 *equal = '\0';
-                char *key = g_strstrip(start);
-                char *value = g_strstrip(equal + 1);
+                char *key = g_strdup(g_strstrip(start));
```

Comment:
Originally, I'd put the quote-handling code in `load_ini_file`, but it didn't work correctly; as I didn't realize that some of the logic was in `clean_value`; so then I put it all in one place; `clean_value`.  

Now, `clean_value` always did an internal `g_strdup`. `add_entry` called `g_strdup` for the key, and `clean_value` for the value.

However, I found it necessary to move `clean_value` from `add_entry` to `load_ini_file` so that an error from `clean_value` could inhibit `add_entry` from running.  When I did that, I also moved the `g_strdup(key)` for symmetry.